downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc

```diff
@@ -10,10 +10,10 @@ Follow the procedures below to configure SSO with Azure Active Directory (Azure
 . In a browser, navigate to the Automation controller URL and log in using the following credentials:
   * Username: `admin`
   * Password: Use the _Administrator password_ you provided when you deployed the {PlatformNameShort} application.
-. In the Automation controller console, click *Settings* in the menu options.
-. Click *Miscellaneous System settings* under the *System* settings.
-. Click *Edit*. Enter the Automation controller URL in the Base URl of the service field.
-. Click *Save*.
+. In the Automation controller console, click btn:[Settings] in the menu options.
```

Comment:
It looks like Settings is actually a menu item and should be changed to menu:Settings[].